packages/react-composites/tests/browser/callwithchat/app/index.tsx

```diff
@@ -33,31 +33,25 @@ const threadId = verifyParamExists(params.threadId, 'threadId');
 initializeIcons();
 
 function App(): JSX.Element {
-  const [callWithChatAdapter, setCallWithChatAdapter] = useState<CallWithChatAdapter>(undefined);
-
-  useEffect(() => {
-    const initialize = async (): Promise<void> => {
-      const credential = new AzureCommunicationTokenCredential(token);
-      const adapter = await createAzureCommunicationCallWithChatAdapter({
-        userId: fromFlatCommunicationIdentifier(userId) as CommunicationUserIdentifier,
-        displayName,
-        credential,
-        endpoint,
-        locator: {
-          callLocator: { groupId: groupId },
-          chatThreadId: threadId
-        }
-      });
-      setCallWithChatAdapter(wrapAdapterForTests(adapter));
-    };
-
-    initialize();
-
-    return () => {
-      callWithChatAdapter && callWithChatAdapter.dispose();
-    };
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  const userIdArg = useMemo(() => fromFlatCommunicationIdentifier(userId) as CommunicationUserIdentifier, []);
```

Comment:
We've mentioned we wanted to useMemo in the jsdoc of useAzureCommunicationCallWithChatAdapter, but would be nice to still add some comments here of why we call "useMemo", I believe there would be a lot developers only read sample code without looking at api doc (like me sometimes)

Reply:
Ohh, this is test code not code snippet, ignore this